docusaurus/docs/Android/04-compose/04-message-components/03-message-list.mdx

```diff
@@ -3,50 +3,65 @@
 The `MessageList` component is a crucial part when building a chat experience. We support two versions of the `MessageList` component:
 
 * **Bound**: This version binds itself to the `MessageListViewModel` and loads all the required data. It also connects single and long item tap, pagination, and bottom reached events to the `ViewModel`.
-* **Stateless**: This is a stateless version of the list, which doesn't know about the `ViewModel` and depends on pure state from external sources, to render its UI.
+* **Stateless**: This is a stateless version of the list which doesn't know about the `ViewModel` and depends on pure state from external sources in order to render its UI.
```

Comment:
I'm the worst tech writer ever, but what do you think about replacing `in order to` with just `to`. Less words and the meaning remains the same?

Reply:
This is a valid suggestion, I will replace it with to.

Reply:
Yeah that's what we should do! :]

Good point. All of these long forms can be usually shortened to `to`

"So that we can build a ...." -> "To build a..." is another example.